crmsh/ui_cluster.py

```diff
@@ -208,7 +208,7 @@ def _enable_disable_common(self, context, *args):
         node_list = parse_option_for_nodes(context, *args)
         action = context.get_command_name()
         utils.cluster_run_cmd("systemctl {} pacemaker.service".format(action), node_list)
-        if utils.is_qdevice_configured():
+        if utils.service_is_available("corosync-qdevice.service"):
```

Comment:
Blindly disable qdevice is okay. But, enable it even without the qdevice configuration won't be intuitive I think.